src/com/missafternoon/budjen/transactions/DateConverter.java

```diff
@@ -1,3 +1,5 @@
+package com.missafternoon.budjen.transactions;
```

Comment:
This class doesn't actually have anything to do with `transactions`. Try and come up with another package that can hold this class - maybe also the `Currency` class. Failing that, put them in a `utils` package.